pages/rops/update/content/index.js

```diff
@@ -105,7 +105,7 @@ Octopus and squid should be counted as free feeding from when they hatch, and cu
 Select all that apply`,
       options: {
         'uk-licenced': {
-          label: 'In the UK at licenced establishment',
+          label: 'In the UK at a licenced establishment',
```

Comment:
I think this should also be `licensed`

Reply:
I was wondering that but the ticket said licenced